integration_test/test_bundle.dart

```diff
@@ -0,0 +1,66 @@
+// ignore_for_file: invalid_use_of_internal_member,
```

Comment:
`patrol_cli` on `develop` branch now automatically generates this file when you `test` or  `build`. (`patrol develop` is currently broken)

To install Patrol CLI from the `develop` branch, run:

```console
dart pub global activate \
  --source git https://github.com/leancodepl/patrol.git \
  --git-path packages/patrol_cli \
  --git-ref develop
```